compiler/ballerina-parser/src/test/resources/tree/separated_node_list_modify_all_nodes_assert.bal

```diff
@@ -0,0 +1 @@
+inta =[a  ,b  ,c];
```

Comment:
Shouldn't the WhiteSpaceMinutiaeRemover remove all the whitespaces?

Reply:
it only removes the white space from all tokens not from nodes like SimpleNamedReferenceNode.